bin/bin

```diff
@@ -0,0 +1 @@
+bin
```

Comment:
I have mistake 😢 Ok, I'll fix it tomorrow.
Because I'm half asleep . now time is 0:47 in Japan. I need to sleep soon . .